### ISSUE TITLE
securedrop-admin (setup) fails in Tails 4.0-rc1

### ISSUE THREAD

**Issue by emkll** [freedomofpress/securedrop#4910]
## Description

Running `./securedrop-admin setup` in tails 4.0-rc1 (upgraded from 3.16) returns the following error:

```
INFO: Virtualenv already exists, not creating
INFO: Checking Python dependencies for securedrop-admin
ERROR: Failed to install pip dependencies. Check network connection and try again.
```
This was done in VMs, will try to reproduce on hardware.

## Steps to Reproduce

1. Upgrade Tails device from 3.16 to 4.0 (Boot into 4.0-rc1 drive and clone to upgrade 3.16 drive)
2. Boot into newly-upgraded tails drive
3. Verify that the internet is working (tor is bootstrapped, you can reach an external website over tor)
4. check out `1.1.0~rc2` tag
5. Run `./securedrop-admin setup` in ~/Persistent/securedrop
6. Observe error

## Expected Behavior

Securedrop-admin should run and the dependencies should be installed.

## Actual Behavior

Securedrop-admin fails and returns an error, the dependencies are not installed

## Comments

Per https://github.com/freedomofpress/securedrop/pull/4852/files#diff-b5e536cc161fcc0d62e661b4d6eae381R70-R73

When running the commands locally, I get
* `lsb_release --id --short` returns `Debian`
* `uname -a` returns `Linux amnesia 5.3.0-trunk-amd64 #1 SMF Debian 5.3.2-1~exp1 (2019-10-02) x86_64 GNU/Linux`

When i run ./securedrop-admin with no parameter, I get:
```
amnesia@amnesia:~/Persistent/securedrop$ ./securedrop-admin help
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ImportError: No module named 'encodings'

Current thread 0x00007cf687450740 (most recent call first):
Aborted
```

**Comment by emkll** [comment]
Removing the existing admin virtualenv in `~/Persistent/securedrop/admin/.venv3` resolves the issue for me locally. 

It seems like Buster/tails4-specific dependencies are required to build a buster-compatible venvs. If the virtualenv created in 3.x (stretch) it doesn't seem to work in 4.x (buster).

**Comment by kushaldas** [comment]
>It seems like Buster/tails4-specific dependencies are required to build a buster-compatible venvs. If the virtualenv created in 3.x (stretch) it doesn't seem to work in 4.x (buster).

Yes, this is due to different Python versions, 3.5 in stretch and 3.7 in Buster.

**Comment by kushaldas** [comment]
As we discussed over chat, @emkll is going to provide another PR with bash script change to detect new OS version + Python version change, this will help to recreate the environment if required.
